feature/bgp/policybase/otg_tests/nested_policies/nested_policies_test.go

```diff
@@ -234,146 +235,220 @@ func TestBGPNestedPolicies(t *testing.T) {
 }
 
 // configureImportRoutingPolicy configures the dut for IPv4 BGP nested import policy test.
-func configureImportRoutingPolicy(t *testing.T, dut *ondatra.DUTDevice) {
+func configureImportRoutingPolicy(t *testing.T, dut *ondatra.DUTDevice, operation string) {
 	batch := &gnmi.SetBatch{}
 	root := &oc.Root{}
 	rp := root.GetOrCreateRoutingPolicy()
 
-	// Configure a route-policy to set the local preference.
-	pdef1 := rp.GetOrCreatePolicyDefinition(v4LPPolicy)
-	stmt1, err := pdef1.AppendNewStatement(v4LPStatement)
+	// Configure PERMIT-ALL policy
+	pdef := rp.GetOrCreatePolicyDefinition("PERMIT-ALL")
```

Comment:
Please have the policy name "PERMIT-ALL" defined as a const 
same for newstatement 20

Reply:
Updated. Please check.